hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -200,12 +214,12 @@ genCloseMutation (tx, _utxo) =
         , contestationDeadline
         }
     st -> error $ "unexpected state " <> show st
-
-  mutateClosedContestationDeadline :: Gen (TxOut CtxTx)
-  mutateClosedContestationDeadline = do
+  -- In case contestation period param is 'Nothing' we will generate arbitrary value
+  mutateClosedContestationDeadline :: Maybe Integer -> Gen (TxOut CtxTx)
```

Comment:
Instead of passing `Maybe` (and conditionally switching logic inside the funciton) I would recommend to pass the `Integer` and generate it outside.

Reply:
Still relevant (also the comment above). But this is a "Could".